README.md

```diff
@@ -45,15 +45,17 @@ In the case you only want to evaluate only one or more models you can use the `-
 Executing the following output:
 
 ```bash
-eval-dev-quality evaluate --model=openrouter/anthropic/claude-3-opus
+eval-dev-quality evaluate --model=openrouter/meta-llama/llama-3-70b-instruct
 ```
 
-Should return an evaluation log like this:
+Should return an evaluation log similar to this:
```

Comment:
nit

since the log grew even more maybe use a `<details>`
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections

Reply:
Great idea!